test/sequential/issuance.issuanceConfig.test.ts

```diff
@@ -0,0 +1,261 @@
+  await SequencerStaking.setupASequencer(testUser, "Ethereum");
+  testUser.addAsset(GASP_ASSET_ID);
+
+  const { disputeEndBlockNumber: disputeEndBlockNumber1 } =
+    await createAnUpdate(testUser, "Ethereum");
+  const rewardsSessionNumber1 = await getSessionIndex();
+  const registrationBlock1 = disputeEndBlockNumber1 + 1;
+  await waitBlockNumber(registrationBlock1.toString(), disputePeriodLength * 2);
+  await waitForSessionN(rewardsSessionNumber1 + 2);
+  const rewardInfo1 = await SequencerStaking.roundSequencerRewardInfo(
+    testUser.keyRingPair.address,
+    rewardsSessionNumber1,
+  );
+  const payoutEvent1 = await Sudo.asSudoFinalized(
+    Sudo.sudoAs(
+      testUser,
+      SequencerStaking.payoutRewards(testUser.keyRingPair.address, 2),
+    ),
+  );
+  expectMGAExtrinsicSuDidSuccess(payoutEvent1);
+  const filteredEvent1 = await filterAndStringifyFirstEvent(
+    payoutEvent1,
+    "Rewarded",
+  );
+  const sequencerRewards1 = stringToBN(filteredEvent1[2]);
+  expect(rewardInfo1).bnEqual(sequencerRewards1);
+
+  await Sudo.batchAsSudoFinalized(await Issuance.setIssuanceConfig(40, 40, 20));
+  await waitForSessionN((await getSessionIndex()) + 1);
+
+  const [testUser2] = setupUsers();
+  await SequencerStaking.removeAddedSequencers();
+  await SequencerStaking.setupASequencer(testUser2, "Ethereum");
+
+  const { disputeEndBlockNumber: disputeEndBlockNumber2 } =
+    await createAnUpdate(testUser2, "Ethereum");
+  const rewardsSessionNumber2 = await getSessionIndex();
+  const registrationBlock2 = disputeEndBlockNumber2 + 1;
+  await waitBlockNumber(registrationBlock2.toString(), disputePeriodLength * 2);
+  await waitForSessionN(rewardsSessionNumber2 + 2);
+  const rewardInfo2 = await SequencerStaking.roundSequencerRewardInfo(
+    testUser2.keyRingPair.address,
+    rewardsSessionNumber2,
+  );
+  const payoutEvent2 = await Sudo.asSudoFinalized(
+    Sudo.sudoAs(
+      testUser2,
+      SequencerStaking.payoutRewards(testUser2.keyRingPair.address, 2),
+    ),
+  );
+  expectMGAExtrinsicSuDidSuccess(payoutEvent2);
+  const filteredEvent2 = await filterAndStringifyFirstEvent(
+    payoutEvent2,
+    "Rewarded",
+  );
+  const sequencerRewards2 = stringToBN(filteredEvent2[2]);
+  expect(rewardInfo2).bnEqual(sequencerRewards2);
+  expect(sequencerRewards1).bnEqual(sequencerRewards2.muln(2));
+});
+
+test("Compare amount of parachainStaking.Rewarded for 2 difference configuration", async () => {
+  await waitForSessionN((await getSessionIndex()) + 3);
+  const firstEvent = await waitUntilUserCollatorRewarded(sudo);
+  const rewardAmount1 = await getRewardsAmount(firstEvent);
+
+  await Sudo.batchAsSudoFinalized(await Issuance.setIssuanceConfig(45, 10, 45));
+  await waitForSessionN((await getSessionIndex()) + 3);
+
+  const secondEvent = await waitUntilUserCollatorRewarded(sudo);
+  const rewardAmount2 = await getRewardsAmount(secondEvent);
+
+  expect(rewardAmount2).bnEqual(rewardAmount1.divn(2));
```

Comment:
this can be flaky, thre can be situations where some collaltor produces more blocks than the other, and rewards can  be affected. 
and how  await Sudo.batchAsSudoFinalized(await Issuance.setIssuanceConfig(40, 20, 40)); from setup to  await Sudo.batchAsSudoFinalized(await Issuance.setIssuanceConfig(45, 10, 45)); makes a *2 change?

Reply:
Done